alpha/scripts/kava/kava_plays_views_sync.sh

```diff
@@ -4,6 +4,9 @@
 echo `date`
 
 echo "get plays/views"
+TEMP_FILE=/tmp/playsviewsDump.txt
```

Comment:
Please set default TEMP to Kaltura's temp:
TEMP_FILE=$BASE_DIR/tmp/playsviewsDump.txt